crates/chaincash_transactions/src/boxes.rs

```diff
@@ -0,0 +1,75 @@
+use ergo_lib::{
```

Comment:
Move `specs` to `offchain` repo, now that we have a service crate there shouldn't be any circular dependency between `store` <-> `offchain`